bench/Chainweb/Pact/Backend/Bench.hs

```diff
@@ -103,7 +103,7 @@ cpWithBench torun =
     name = "batchedCheckpointer"
     cid = unsafeChainId 0
 
-    initialBlockState = initBlockState $ genesisHeight v cid
+    initialBlockState = initBlockState (1024^(2::Int) {- 1MiB -}) $ genesisHeight v cid
```

Comment:
I was going to suggest using `mega` from `Chainweb.Utils` but those are SI. Maybe we should make powers-of-2 versions for use cases like this instead of having to comment the size.

Reply:
Also, any reason for this size for the benchmark?

Reply:
No particular reason, I just picked something. We could just use the prod defaults. I can think of pros and cons for both larger or smaller values. So not sure.